### ISSUE TITLE
Receiving error when exporting Document Templates - TypeError: Cannot read property 'replace' of null

### ISSUE THREAD

**Issue by SamuelMoy** [vlocityinc/vlocity_build#103]
C:\Users\xxxx\AppData\Roaming\npm\node_modules\vlocity\node_modules\sass.js\dist\sass.sync.js:61
var Module;if(!Module)Module=(typeof Module!=="undefined"?Module:null)||{};var moduleOverrides={};for(var key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}var ENVIRONMENT_IS_WEB=false;var ENVIRONMENT_IS_WORKER=false;var ENVIRONMEN
T_IS_NODE=false;var ENVIRONMENT_IS_SHELL=false;if(Module["ENVIRONMENT"]){if(Module["ENVIRONMENT"]==="WEB"){ENVIRONMENT_IS_WEB=true}else if(Module["ENVIRONMENT"]==="WORKER"){ENVIRONMENT_IS_WORKER=true}else if(Module["ENVIRONMENT"]==="NODE"){ENVIRONMENT_IS_NODE=t
rue}else if(Module["ENVIRONMENT"]==="SHELL"){ENVIRONMENT_IS_SHELL=true}else{throw new Error("The provided Module['ENVIRONMENT'] value is not valid. It must be one of: WEB|WORKER|NODE|SHELL.")}}else{ENVIRONMENT_IS_WEB=typeof window==="object";ENVIRONMENT_IS_WORK
ER=typeof importScripts==="function";ENVIRONMENT_IS_NODE=typeof process==="object"&&typeof require==="function"&&!ENVIRONMENT_IS_WEB&&!ENVI

**Comment by rutlabaga** [comment]
Can you include more of the error output? There should be something below and above this.

**Comment by SamuelMoy** [comment]
This is another similar error I think. But not specific with Document Templates.

![image](https://user-images.githubusercontent.com/9741637/47549106-52a90680-d8fb-11e8-90cb-1b8790fb8e61.png)

**Comment by rutlabaga** [comment]
Unfortunately you are cutting the relevant parts of the error messages off. You should include everything from the log if possible.

**Comment by thvd** [comment]
I experience the same error, and have captured the logging output:
```
TypeError: Cannot read property 'replace' of null
    at module.exports (x/node_modules/unidecode/unidecode.js:20:14)
    at module.exports.DataPacksExpand.generateFolderOrFilename (x/node_modules/vlocity/lib/datapacksexpand.js:18:29)
    at module.exports.DataPacksExpand.generateFolderPath (x/node_modules/vlocity/lib/datapacksexpand.js:13:32)
    at module.exports.DataPacksExpand.generateFilepath (x/node_modules/vlocity/lib/datapacksexpand.js:43:17)
    at x/node_modules/vlocity/lib/datapacksexpand.js:685:51
    at Array.forEach (<anonymous>)
    at module.exports.DataPacksExpand.processDataPack (x/node_modules/vlocity/lib/datapacksexpand.js:665:61)
    at x/node_modules/vlocity/lib/datapacksexpand.js:997:22
    at Array.forEach (<anonymous>)
    at module.exports.DataPacksExpand.expand (x/node_modules/vlocity/lib/datapacksexpand.js:995:32)


```

**Comment by rutlabaga** [comment]
Fixed in v1.7.9.
